app/forms/choose_role_form.rb

```diff
@@ -63,33 +60,18 @@ def has_no_role
 private
 
   def rejected_roles
-    %w[induction_coordinator early_career_teacher mentor teacher].freeze
+    %w[early_career_teacher mentor teacher].freeze
```

Comment:
Weird `induction_coordinator` was rejected

Reply:
It was added in again in the subsequent code (if the user was a SIT) as this was trying to make sure only one "school" role was selected, looks weird now but it was doing the right thing